test/test_1_ssl.py

```diff
@@ -13,11 +13,10 @@
         else:
             from urllib.request import urlopen
 
-        if sys.version_info[0:2] == (3, 3):
-            data = urlopen("https://www.nist.gov")
-        else:
-            context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
-            data = urlopen("https://www.nist.gov", context=context)
+        context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
+        data = urlopen("https://www.nist.gov", context=context)
+        data = urlopen("https://raw.githubusercontent.com/joerick/cibuildwheel/master/CI.md", context=context)
```

Comment:
Do we need to assert something for the received data? E.g. just that it's not empty, or so?

Reply:
maybe we should, but then we should have some file with some fixed content.

Reply:
Not per se. You could just test if something non-empty was downloaded? Or take away the `data =`, maybe? It just feels weird, all these useless assignments.